pkg/network/tracer/tracer.go

```diff
@@ -142,14 +141,8 @@ func newTracer(cfg *config.Config) (_ *Tracer, reterr error) {
 			}
 			log.Warnf("%s. NPM is explicitly enabled, so system-probe will continue with only NPM features enabled.", errStr)
 			cfg.EnableHTTPMonitoring = false
-			cfg.EnableHTTP2Monitoring = false
```

Comment:
Is this also being set elsewhere?

Reply:
it is false by default and the customer have to explicitly enable the feature

Reply:
Do we want to explicitly set this config to `false` here to be consistent with the behavior here? https://github.com/DataDog/datadog-agent/blob/main/pkg/network/protocols/http2/protocol.go#L102

Reply:
no, the behavior I'm removing now was added by mistake (copying http behavior)
so disabling the configuration won't make any change, as https://github.com/DataDog/datadog-agent/blob/main/pkg/network/protocols/http2/protocol.go#L102 is the only location that uses that config